code/game/objects/structures/fullwindow.dm

```diff
@@ -12,12 +12,15 @@
 	mouse_opacity = 2 // Complete opacity //What in the name of everything is this variable ?
 	layer = 3.21 // Windows are at 3.2.
 
+	penetration_dampening = 1
+
+
 	cracked_base = "fcrack"
 
 /obj/structure/window/full/New(loc)
 
 	..(loc)
-	flags &= ~ON_BORDER
+	flags |= ON_BORDER
```

Comment:
This is removing the on_border flag currently when you spawn a fulltile window, isn't it?

I thought that was intended?